axelrod/strategies/handshake.py

```diff
@@ -0,0 +1,33 @@
+from axelrod import Actions, Player, Game
+
+C, D = Actions.C, Actions.D
+
+
+class Handshake(Player):
+    """Starts with C, D. If the opponent plays the same way, cooperate forever,
+    else defect forever."""
+
+    name = 'Handshake'
+    classifier = {
+        'memory_depth': float('inf'),  # Long memory
+        'stochastic': False,
+        'makes_use_of': set(),
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def __init__(self):
+        Player.__init__(self)
+        self.initial_plays = [C, D]
```

Comment:
I don't understand the drawback of parametrising this. This would be:

```
+    def __init__(self, initial_plays=[C, D]):
+        Player.__init__(self)
+        self.initial_plays = initial_plays
```

Seems simple enough and harmless, also immediately means that it can be used for to see effect of initial plays. Same comment for above.